guides/serve.md

```diff
@@ -0,0 +1,60 @@
+---
+summary: Get up and running with `denoflare serve`
+---
+
+# Start a New Worker
+
+Follow these steps to setup `denoflare serve` to run from a config file.  This assumes you've already [installed the denoflare CLI](/cli).
+
+1. Create a project folder. Add an entry script:
+
+    ```ts
+    // index.ts
+    export default {
+        async fetch(request: Request, env: any) {
+            try {
+                return new Response('hi')
+            } catch (e) {
+                return new Response(e.message)
+            }
+        },
+    }
+    ```
+
+2. Create a config file:
+
+    ```jsonc
+    // denoflare.jsonc
+    {
+        "$schema": "https://raw.githubusercontent.com/skymethod/denoflare/v0.3.3/common/config.schema.json",
+        "scripts": {
+            "local": {
```

Comment:
Maybe "hello-world-local" or "hello-local"?  I'm afraid "local" makes it look like a category or designation for all local scripts, rather than simply an identifier

Reply:
Oh good point.  I'll update that now